lms/envs/labster.py

```diff
@@ -2,6 +2,9 @@
 import urlparse
 
 
+# Regions of Labster
+REGIONS = ENV_TOKENS.get('LABSTER_REGIONS', {})
```

Comment:
@lukyfebrianto please create a tasl for DevOps with example of data that needds ot be added to the edx settings

Reply:
https://liv-it.atlassian.net/browse/OPS-523